### PR TITLE
Add post for improvement of error messages

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

File: _posts/development/2022-04-20-scm-integration-better-error-messages.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{% include partials/_series-of-posts-about-scm-integration.md %}
+
+# The Workflow Runs Page Helps You Understand
+
+You can find the list of workflow runs on _Profile > Manage Your Tokens_.
+
+The workflow runs are there to help you understand what's going on with every
+workflow you run: its state, the sources and artifacts involved etc. But
+probably the most interesting piece of information is in the Response tab,
+where you can detect if everything went well or you can read a descriptive
```
Comment (danidoni) [inline _posts/development/2022-04-20-scm-integration-better-error-messages.md:25]:
```suggestion
where you can check if everything went well or you can read a descriptive
```

File: _posts/development/2022-04-20-scm-integration-better-error-messages.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+error if something went wrong.
+
+[Screenshot]
```
Comment (hennevogel) [inline _posts/development/2022-04-20-scm-integration-better-error-messages.md:28]: Screenshot is missing

File: _posts/development/2022-04-20-scm-integration-better-error-messages.md (hunk 1 above)
Comment (saraycp) [reply _posts/development/2022-04-20-scm-integration-better-error-messages.md]: done

File: _posts/development/2022-04-20-scm-integration-better-error-messages.md (hunk 2 above)
Comment (saraycp) [reply _posts/development/2022-04-20-scm-integration-better-error-messages.md]: Screenshot added.

**Comment by saraycp** [comment]
@danidoni @hennevogel I already addressed your feedback.

**Review by danidoni — APPROVED** [review]

**Pull request merged**